include/fmt/base.h

```diff
@@ -2223,7 +2223,7 @@ struct locale_ref {
  public:
   constexpr locale_ref() : locale_(nullptr) {}
   template <typename Locale> explicit locale_ref(const Locale& loc);
-  explicit operator bool() const noexcept { return locale_ != nullptr; }
+  FMT_INLINE explicit operator bool() const noexcept { return locale_ != nullptr; }
```

Comment:
Here and elsewhere, please use `inline` instead of `FMT_INLINE` since the latter expands to always inline in optimized mode.